doc/stdlib/ffi.rst

```diff
@@ -248,3 +248,21 @@
 .. data:: Cdouble
 
    Equivalent to the native ``double`` c-type (Float64)
+
+****************
+ LLVM Interface
+****************
+
+.. function:: llvmcall(IR::String, ReturnType, (ArgumentType1, ...), ArgumentValue1, ...)
+
+   Call LLVM IR string in the first argument. As in an LLVM function ``define``
+   block, arguments are available as consecutive unnamed SSA variables (%0, %1, etc.).
+
+   Note that the argument type tuple must be a literal tuple, and not a tuple-valued variable or expression.
+
+   Each ``ArgumentValue`` to ``llvmcall`` will be converted to the corresponding ``ArgumentType``,
+   by automatic insertion of calls to ``unsafe_convert(ArgumentType, cconvert(ArgumentType, ArgumentValue))``.
+   (see also the documentation for each of these functions for further details).
+   In most cases, this simply results in a call to ``convert(ArgumentType, ArgumentValue)``.
+
+   See ``test/llvmcall.jl`` for usage examples.
```

Comment:
Does
`` `test/llvmcall.jl 
<https://github.com/JuliaLang/julia/blob/master/test/llvmcall.jl>`_`` work in this context?